cli.js

```diff
@@ -33,6 +34,7 @@ Options:
   -h --help                 Display help and usage details.
   -l --limit                Limit the number of items to display (defaults to 15).
   -k --keep-open            Wether or not to keep the table open after selecting an item (defaults to false).
+  --latest                  Sort the list by latest entries
```

Comment:
@erickbelfy Please, add enough whitespace here so `--latest` is aligned to left with `--keep-open`. Like `--module` here: https://github.com/rafaelrinaldi/npm-downloads#usage